terraform/vars/skylight/main.tf

```diff
@@ -24,3 +24,13 @@ module "network" {
   source = "../../modules/network"
 }
 
+module "fhir" {
+  source         = "../../modules/fhir"
+  region         = "us-west1"
+  time_zone      = "UTC"
+  dataset_name   = "PHDI_DATASET-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
+  fhirstore_name = "PHDI_FHIRSTORE-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
```

Comment:
What happens if two instances of the pipeline are spun up in a single day? I think it is foreseeable that one user might create dev and prod environments in close succession. This is why I was proposing  `PHDI_DATASET-${time_static.current_date.unix}` which specifies down to the second level.

Reply:
Good point.  I was thinking about this also and wasn't sure about the use case.  However, it's an easy change to make and better to be safe than sorry.  I'll adjust.